pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.pytest.ini_options]
+addopts = "--doctest-modules"
+testpaths = [
```

Comment:
does this mean we'll potentially test doctests in `xdem/tests` as well?

Reply:
Yes!

Reply:
The testpaths is there because the `examples/` directory will otherwise be run, and I have no idea why, but then it fails! (there are no doctests there, but it still fails...)

Reply:
Weird one!